### ISSUE TITLE
Mean wait time vs. mean headway

### ISSUE THREAD

**Issue by smmaurer** [UDST/urbanaccess#76]
When UrbanAccess creates connector edges for agents moving from the base network to the transit network, the default impedance is the mean headway at a given transit stop. 

This happens in `urbanaccess.integrate_network()`, and current options are mean, std, min, and max, defined here: [network.py#L103](https://github.com/UDST/urbanaccess/blob/dev/urbanaccess/network.py#L103).

I'd like to propose adding an option for _half_ of the mean, which would be a better approximation for the average expected wait time. It's also possible to achieve this through post-processing of the integrated network, but automatic calculation of it might be nice!

This is related to issues #36 and #51, i think.

**Comment by sablanchard** [comment]
Agreed @smmaurer . This would be a good addition as another option to use in the parameter along side the existing ones. 

Would also be nice if while this can be addressed we open this up to also add an option here to pass a custom function to calculate the headway as well.
